src/client/localplayer.h

```diff
@@ -149,7 +149,7 @@ class LocalPlayer : public Player
 
 	bool getAutojump() const { return m_autojump; }
 
-	bool isDead() const { return hp <= 0; }
+	bool isDead() const;
```

Comment:
Moved the function's definition to the cpp file as it now references `GenericCAO::getHP` and `GenericCAO::getGroups`.